packages/clay-layout/src/__tests__/index.tsx

```diff
@@ -96,4 +96,18 @@ describe('ClayLayout', () => {
 
 		expect(container).toMatchSnapshot();
 	});
+
+	it('removes size class when prop is set to false', () => {
+		const {container} = render(
+			<>
+				<ClayLayout.ContainerFluid size={false}>
+					{'test'}
+				</ClayLayout.ContainerFluid>
+			</>
+		);
+
+		expect(
+			container.querySelector('.container-fluid')!.classList
+		).toMatchSnapshot();
```

Comment:
It seems a shame to have to look at a snapshot file just to see what we expect the class list to look like (ie. a single class, `.container-fluid`). The trouble with snapshots is generally that they say too much (which means it can be hard to know whether an update to them is legitimate or evidence of breakage); but this one suffers from the opposite problem — that it doesn't say enough to justify being a snapshot. Can we inline this and avoid the snapshot?

Reply:
Good call, just updated